mypyc/genops.py

```diff
@@ -192,13 +198,55 @@ def build_ir(modules: List[MypyFile],
 
     # Compute vtables.
     for cir in class_irs:
-        compute_vtable(cir)
+        if cir.is_ext_class:
+            compute_vtable(cir)
 
     errors.flush_errors()
 
     return mapper.literals, result, errors.num_errors
 
 
+def is_extension_class(cdef: ClassDef) -> bool:
+    if any(not (isinstance(d, RefExpr) and d.fullname == 'mypy_extensions.trait')
+           for d in cdef.decorators):
+        return False
+    elif cdef.info.metaclass_type and cdef.info.metaclass_type.type.fullname() == 'enum.EnumMeta':
```

Comment:
I think we want to do this for *any* metaclass

Reply:
(Excep `TypingMeta`, `GenericMeta`, `ABCMeta`, but I think those get stripped out?)